pkg/webservice/handlers.go

```diff
@@ -481,9 +482,9 @@ func getPartitions(w http.ResponseWriter, r *http.Request) {
 }
 
 func getPartitionQueues(w http.ResponseWriter, r *http.Request) {
-	vars := mux.Vars(r)
+	vars := httprouter.ParamsFromContext(r.Context())
```

Comment:
httprouter.ParamsFromContext can return nil: https://github.com/julienschmidt/httprouter/blob/master/router.go#L118-L119
Should we not handle that or are we 100% sure that can never happen?

Reply:
Right, we can't 100% sure, i will address this change.